libraries/AP_HAL_ChibiOS/hwdef/SPRacingH7/hwdef.dat

```diff
@@ -0,0 +1,169 @@
+# hw definition file for processing by chibios_pins.py
+# for SP/Racing Extreme H7 hardware.
+# thanks to betaflight for pin information
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
```

Comment:
Why is the MCU spec'ed as H743 and not H750

Reply:
@mluessi only just provided a definition script, this was a shortcut by me before that was available. I'll fix when I get back to looking at this.

Reply:
Fixed